new/LIT6688TeachingMysteries.xml

```diff
@@ -63,6 +63,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <listRelation>
                     <relation name="saws:isRelatedTo" active="LIT6688TeachingMysteries" passive="LIT2444Temher"/>
                     <relation name="saws:isRelatedTo" active="LIT6688TeachingMysteries" passive="LIT3978Elmastoagya"/>
+                      <relation name="saws:isRelatedTo" active="LIT6688TeachingMysteries" passive="LIT1586Hayman"/>
```

Comment:
for LIT1586Hayman should be ecrm:CLP46i_may_form_part_of

Reply:
ok